src/Orc.Controls/Controls/LogViewer/Views/LogViewerControl.xaml.cs

```diff
@@ -504,10 +504,6 @@ private void ClearScreen()
             rtb.Visibility = Visibility.Visible;
 
             var oldDoc = rtb.Document;
-            if (oldDoc is null)
```

Comment:
why remove this null check? It can never be null?

Reply:
Sure, Document throwing Argument exception if you try to set this null